src/routes/index.tsx

```diff
@@ -11,6 +12,7 @@ export const routes: RouteDefinition[] = [
     { path: '/flashFirmware', component: FlashFirmware },
     { path: '/network', component: NetworkConfigurator },
     { path: '/configureBoard', component: BoardConfigurator },
```

Comment:
since we're going with the wizard naming scheme, we might want to be consistent and rename our old configurator to a wizard as well, what do you think? 

It's more of a consistency thing, doesn't block the merge in any way